packages/ceramic-cli/src/ceramic-cli-utils.ts

```diff
@@ -381,7 +383,7 @@ export class CeramicCliUtils {
      * @private
      */
     static _generateSeed(): string {
-        const seed = '0x' + Buffer.from(randomBytes(32)).toString('hex')
+        const seed = u8a.toString(randomBytes(32))
```

Comment:
is this what IDW uses? Should it not be a uint8array?

Reply:
this is just because of the serialization - saving it to `config.json`. I think that we can remove this method now 👍